### PR TITLE
Docs: document artiq_coreanalyzer usage.

### PR DESCRIPTION
**Pull request by drewrisinger**
See #1232, #206, #550.

Signed-off-by: Drew Risinger <drewrisinger@users.noreply.github.com>

### CODE REVIEW

**Comment by cjbe** [comment]
It might be worth mentioning the other functionality of the core analyser. I have found the "print decoded" output very useful for debugging in the past - this shows all RTIO events, including input events etc. For example, this allows one to see the timestamps of input trigger events, or input photon counts.

**Comment by drewrisinger** [comment]
Great, I'd welcome more info about it. I honestly have no clue what this tool does.

File: doc/manual/utilities.rst [hunk 1]
```diff
@@ -172,6 +172,8 @@ You do not need to remove a record in order to change its value, just overwrite
 Core device RTIO analyzer tool
 ------------------------------
 
+:mod:`~artiq.frontend.artiq_coreanalyzer` is a tool to convert core device RTIO logs to VCD waveform files that are readable by third-party tools such as GtkWave. This tool extracts pre-recorded data from an ARTIQ core device buffer, and converts it to a standard VCD file format. See :ref:`rtio-analyzer-example` for an example, or :mod:`artiq.test.coredevice.test_analyzer` for a relevant unit test.
```
Comment (sbourdeauducq) [inline doc/manual/utilities.rst]: It can also read from a file using ``-r``.

Comment (drewrisinger) [reply doc/manual/utilities.rst]: Done

**Pull request merged**